zebrad/tests/acceptance.rs

```diff
@@ -29,8 +29,13 @@ fn generate_no_args() -> Result<()> {
     let output = child.wait_with_output()?;
     let output = output.assert_success()?;
 
+    // First line
     output.stdout_contains(r"# Default configuration for zebrad.")?;
 
+    // Make sure we have no info message in output
+    let notfound = output.stdout_contains(r"INFO");
+    assert!(notfound.is_err());
+
```

Comment:
Issue: This check doesn't cover all possible log levels.

If the log filter is set at "info" level, we can get logs at these levels:
* `INFO`
* `WARN`
* `ERROR`
* `error` (from the config)
* `panicked`

So we should do a case-insensitive check for any of these strings. (There doesn't seem to be any other common pattern in log messages.)

Suggestion: Let's implement this check as a function, rather than copying the code 5 times.

Reply:
If we do this, we encode the current log output behavior into our test suite, even though it's not really something that we're planning to make stability guarantees about. The reason that we don't want there to be any unrelated messages in the command output is that the command output is supposed to be a valid config file.

So instead of checking for specific messages, we could just check that the output of `generate` is a valid config file, which is the property we actually want.

Reply:
> So instead of checking for specific messages, we could just check that the output of `generate` is a valid config file, which is the property we actually want.

That exact test on `generate` should be covered by ticket #845 and PR #859, not this PR.

This PR is supposed to be about `revhex`, `version`, `help`, and `zebrad` with no arguments.

I'll update ticket #847 and my review to try to make that clearer, and reduce scope here.